src/mbgl/map/map.cpp

```diff
@@ -271,8 +289,8 @@ void Map::resetNorth() {
 
 #pragma mark - Pitch
 
-void Map::setPitch(double pitch) {
-    transform->setPitch(std::min(pitch, 60.0) * M_PI / 180);
+void Map::setPitch(double pitch, const Duration& duration) {
+    transform->setPitch(util::clamp(pitch, 0., 90.) * M_PI / 180, duration);
```

Comment:
The 60 here was intentional. We don't support level of detail loading so if the pitch is set it to 90 it will try to load all the tiles in the world